src/main/java/seedu/utility/Messages.java

```diff
@@ -62,9 +64,9 @@
     public static final String DISPLAY_ADVICE_ERROR = "Sorry there is no advice for you at this moment >.<";
     public static final String CURRENT_CURRENCY_MESSAGE = "You currency setting currently: ";
     public static final String INVALID_DATE_RANGE_MESSAGE = "Make sure your start date is before your end date";
-    public static final String MISSING_PARAMETERS_MESSAGE = "Missing parameters! Use help to view commands again!";
-    
-    public static final String INVALID_EXPENSE_VALUE = "Wow! That's a lot of money."
+    public static final String PARAMETERS_ERROR_MESSAGE = "You have missing or invalid parameters. Use help to view " 
+            + "commands again!";
+    public static final String INVALID_EXPENSE_VALUE = "Wow! That's a lot of money. "
```

Comment:
I'm worried that it isn't clear enough from this message that he has exceeded our max limit for entries.

Reply:
Alright we shall look into that soon.